### DISCUSSION TITLE
Can I sell in "part quantities" such as 0.5

### DISCUSSION THREAD

**Discussion by kevinruscoe** [swellstores/community#97] [Help] [answered]
Is is possible to sell things in float quantities? I have a store which sells items by metres (notably 0.25m). Is this possible with Swell?

**Comment by rodrigoddalmeida** [comment, marked as answer]
Hi @kevinruscoe, while we don't support it out-of-the-box, customizing types for fields related to product and stock level quantities is possible, though it would affect the logic for all products - which means it could be hard to maintain (many fields would need to be converted to floats) or make it more complex if you also need to sell products by whole number quantities. 

We could compile a list of models and fields that are used by quantities/stock if that's something you want to pursue.

However, if you are looking for alternatives, it could also be managed with product options (e.g. a 0.25m option that can be ordered multiple times) or by scaling quantity with a different unit internally (i.e. stock is based on a centimeter unit instead:  0.25m = quantity: 25).

**Reply by logeshswell** [reply]
Yes, I would also suggest to manage the quantity internally with different units and it could be a great choice.
